test/stats/watch/stats-and-watch.test.js

```diff
@@ -3,22 +3,26 @@
 const { runWatch } = require('../../utils/test-utils');
 
 describe('stats and watch', () => {
+    beforeAll(() => {
+        process.env.FORCE_COLOR = true;
```

Comment:
It will not work, because we inside `execa`, we should pass it to options, i.e. `await runWatch(__dirname, ['-c', './webpack.config.js', '--color'], { env: { FORCE_COLOR: true } });`

Reply:
resolved